packages/ai/src/ai/common/chat.py

```
@@ -61,17 +63,16 @@ def __init__(self, provider: str, connConfig: Dict[str, Any], bag: Dict[str, Any
 
         # Extract model configuration - these are the core settings that control
         # how the chat driver behaves with respect to token limits
-        self._model = config.get('model')  # Model identifier (e.g., 'gpt-4', 'claude-3')
+        self._model = validate_model_name(config.get('model'))
         self._modelTotalTokens = config.get('modelTotalTokens', 16384)  # Default to 16K if not specified
         self._modelOutputTokens = config.get('modelOutputTokens', 4096)  # Default to 4K if not specified
 
         # We really can't work with a model that has a very small output window
         if self._modelOutputTokens < 1024:
             raise ValueError(f'Model output tokens ({self._modelOutputTokens}) must be at least 1024')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)